### DISCUSSION TITLE
FAKE / SCAM Fooocus websites and accounts

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/Fooocus#2146] [Announcements]
Below websites and social media accounts are **fake** and **NOT** related to us:

* "Fooocus: The Most Advanced AI 2024" fooocus.pro **FAKE website.** Also an image generator cannot be The Most Advanced AI 2024
* "Fooocus Online" fooocus.cc **FAKE website.**
* "twitter/fooocusAI" twitter.com/fooocusAI,  **FAKE twitter account.** We do not have a twitter account
* "Fooocus AI Creations" twitter, @fooocusAI  **FAKE twitter account.** We do not have a twitter account

EDIT @mashb1t:
As there are more and more suspicious websites popping up I've started collecting a list of SCAM URLs.

* https://fooocus.one/
* https://fooocus.pro/
* https://fooocusai.com/
* https://fooocus.cc/
* https://fooocus.co/
* and many more

DO NOT INTERACT WITH THESE WEBSITES OR PAY THEM ANY MONEY!

Fooocus does not have a website. If you nevertheless want to create image online with Fooocus it's recommended to use [Colab](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#colab) (click Open in Colab) or as a paid legit alternative https://rundiffusion.com/ (pay for server instance per hour).

### This GitHub repo is the only official way to get Fooocus. 

*But we are proud of the fact that many people want to fake fooocus rather than faking other software.*

**Comment by mashb1t** [comment]
For reference:
- fooocus.pro: https://github.com/lllyasviel/Fooocus/issues/1908
- BlueMoonAI: https://github.com/lllyasviel/Fooocus/discussions/1855

**Reply by HumblePie1960** [reply]
A response would be most welcome.

**Reply by mashb1t** [reply]
Concerning which topic?

**Reply by HumblePie1960** [reply]
This one ... the one I replied to  ... about whether FooocusPro website.

**Reply by HumblePie1960** [reply]
Sorry I didn't meant to cause this to go off topic.  The question was, it the FooocusPro  website official or not?  How can that be off topic?

**Reply by mashb1t** [reply]
ah, that's what you meant! Yes, we've already discussed that in https://github.com/lllyasviel/Fooocus/issues/1908, so i hope you get why this was a confusing situation.

**Reply by moribundant** [reply]
What's the old saw? Imitation is the sincerest form of flattery 😉.

**Reply by mashb1t** [reply]
@moribundant sure, but, there's a reason why projects have a license and intellectual property theft wihout necessary attribution and reference are not conform with Fooocus GPL v3 license.

**Reply by moribundant** [reply]
@mashb1t: didn't quite get the drift, did you?

**Comment by MT-276** [comment]
### The nerve of these people.

From : https://fooocus.one/
![image](https://github.com/user-attachments/assets/a671f779-960c-4117-a7b3-b875e08e1482)

From : https://fooocus.one/
![image](https://github.com/user-attachments/assets/66dd0f5c-1a1f-49bc-813f-fe5a985b6e24)

Can't you guys do a DMCA takedown ? But I believe that it would take a big chunk of the donations.
